src/components/sidebar/SidebarToc.svelte

```diff
@@ -10,23 +10,29 @@
         hidden?: boolean;
     }[];
 
+    export let minHeadingDepth: number = 1;
     export let maxHeadingDepth: number = 4;
 
+    // filter out headings that are lower than min_heading_depth
+    headings = headings.filter((h) => h.depth >= minHeadingDepth);
+
     // filter out headings that are higher than max_heading_depth
     headings = headings.filter((h) => h.depth <= maxHeadingDepth);
 
-    const min_heading_depth = Math.min(...headings.map((h) => h.depth));
```

Comment:
@mashehu i left this in, as I think it's doing a different thing (despite the name). The config could be set to 1 as default, but the page could only have h3 and down.. So we still need to check the minimum on the page, right?

Reply:
Ah, yes, true. Will merge the two.